pyerrors/fits.py

```diff
@@ -507,32 +584,44 @@ def chisqfunc_residuals(p):
         print('chisquare/d.o.f.:', output.chisquare_by_dof)
 
     if kwargs.get('expected_chisquare') is True:
-        W = np.diag(1 / np.asarray(dy_f))
-        cov = covariance_matrix(y)
-        A = W @ jacobian(func)(fit_result.x, x)
-        P_phi = A @ np.linalg.inv(A.T @ A) @ A.T
-        expected_chisquare = np.trace((np.identity(x.shape[-1]) - P_phi) @ W @ cov @ W)
-        output.chisquare_by_expected_chisquare = chisquare / expected_chisquare
+        if kwargs.get('correlated_fit') is True:
```

Comment:
I would suggest to leave `output.chisquare_by_expected_chisquare` empty in this case